File: src/test/java/sort/SelectionSortTest.java
package sort;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class SelectionSortTest {

    @Test
    void sort() {
        int[] data = {3, 5, 4, 1, 2, 6};

        assertArrayEquals(new int[]{1, 2, 3, 4, 5, 6}, BubbleSort.sort(data));

        int[] data1 = {3, 5, 4, 4, 1, 2, 6};

        assertArrayEquals(new int[]{1, 2, 3, 4, 4, 5, 6}, BubbleSort.sort(data1));
    }
}

File: app/src/test/java/org/lstec/algorithm/sort/SelectionSortTest.kt
package org.lstec.algorithm.sort

import org.junit.Assert
import org.junit.Test

internal class SelectionSortTest {
    @Test
    fun sort() {
        val data = intArrayOf(3, 5, 4, 1, 2, 6)
        Assert.assertArrayEquals(intArrayOf(1, 2, 3, 4, 5, 6), SelectionSort.sort(data))
        val data1 = intArrayOf(3, 5, 4, 4, 1, 2, 6)
        Assert.assertArrayEquals(intArrayOf(1, 2, 3, 4, 4, 5, 6), SelectionSort.sort(data1))
    }
}